lib/manager/index.ts

```diff
@@ -97,12 +97,16 @@ export function getPackageUpdates(
 
 export function extractPackageFile(
   manager: string,
-  content: string,
+  fileContent: string,
   fileName?: string,
   config?: ExtractConfig
 ): Result<PackageFile | null> {
```

Comment:
Shouldn't this be `ExtractPackageFileConfig` argument too?

Reply:
It can extend it by including `manager`

Reply:
Should I create an extended type here or just add the `manager` to the original interface?

Reply:
I think extend would be best option.